lib/rubocop/cop/lint/literal_in_condition.rb

```diff
@@ -65,17 +65,25 @@ def on_case(case_node)
           end
         end
 
+        def on_send(node)
+          case node.method_name
```

Comment:
I think guard clauses are commonly used for this.

Perhaps `SendNode` should have a predicate `#negation_method?`. That would allow a guard clause:

```ruby
return unless node.negation_method?
```

Reply:
Sounds good 👍